Dockerfile

```diff
@@ -70,6 +99,10 @@ RUN cabal update && \
     ihaskell install && \
     rm -fr $(echo ~/.cabal/bin/* | grep -iv ihaskell) ~/.cabal/packages ~/.cabal/share/doc ~/.cabal/setup-exe-cache ~/.cabal/logs
 
+# Spark kernel spec
+RUN mkdir -p $HOME/.ipython/kernels/spark
```

Comment:
Would it be appropriate to call this a Scala kernel instead or do you see this as dissimilar?